### DISCUSSION TITLE
Leveraging the Objeck VM to run other languages?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#366] [General]
Is it possible? Or is the Objeck VM so tied to the Objeck language? Running other languages on a VM that was originally designed for one language is not rare. The JVM comes to mind immediately. It was originally designed for Java, but now there are a bunch of JVM languages available. They are even going further with GraalVM. The Elixir language is also designed to run on the Erlang VM from the beginning.

**Comment by objeck** [comment]
The VM is portable with an instruction set.
